pkg/cmd/roachtest/decommission.go

```diff
@@ -437,6 +437,28 @@ func runDecommissionAcceptance(ctx context.Context, t *test, c *cluster) {
 		t.Fatalf("recommission failed: %v", err)
 	}
 
+	// Verify the --self flag works.
+	t.l.Printf("re-decommissioning first node, from itself\n")
+	selfFlagSupported := true
+	if cmdOutput, err := decommission(ctx, 1, nil, "decommission", "--self"); err != nil {
+		// Until 20.2, --self is not supported.
+		// TODO(knz): Remove this alternative when roachtest does not
+		// test any version lower than 20.2.
```

Comment:
Is this roachtest testing v20.1 below? Where's that happening? (I thought it was only testing current HEAD, I don't see any multi-version machinery hooked up here.)

Reply:
This roachtest tries the `--self` flag and then bails out from that part of the test if the flag is not supported. This will automatically fail-and-ignore for everything 20.1 or below.